content/06.methods.md

```diff
@@ -70,75 +69,72 @@ Tissues were lysed using a Qiagen TissueLyser II (Qiagen) with 2×30 sec at 18Hz
+For PBTA samples, we used four variant callers to call SNVs and indels from panel, WXS, and WGS data: Strelka2 [@doi:10.1038/s41592-018-0051-x], Mutect2 [@doi:10.1101/861054], Lancet [@doi:10.1038/s42003-018-0023-9], and VarDict [@doi:10.1093/nar/gkw227].
+VarDict-only calls were not retained since ~ 39M calls with low VAF were uniquely called and may be potential false positives.
+We used only Strelka2, Mutect2 and Lancet to analyze WXS samples from TCGA.
+TCGA samples were captured using various WXS target capture kits and we downloaded the BED files from the [`GDC portal`](https://api.gdc.cancer.gov/files).
+The manufacturers provided the input interval BED files for both panel and WXS data for PBTA samples.
+We padded all panel and WXS BED files were by 100 bp on each side for Strelka2, Mutect2, and VarDict runs and by 400 bp for the Lancet run.  
 For WGS calling, we utilized the non-padded BROAD Institute interval calling list [`wgs_calling_regions.hg38.interval_list`](https://console.cloud.google.com/storage/browser/_details/genomics-public-data/resources/broad/hg38/v0/wgs_calling_regions.hg38.interval_list), comprised of the full genome minus N bases, unless otherwise noted below.
-Strelka2 [@doi:10/gdwrp4] v2.9.3 was run using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors.
-The final Strelka2 VCF was filtered for PASS variants.
-Mutect2 from GATK v4.1.1.0 was run following Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl].
-The final Mutect2 VCF was filtered for PASS variants.
-To manage memory issues, VarDictJava [@doi:10.1093/nar/gkw227] v1.58 [@url:https://github.com/AstraZeneca-NGS/VarDictJava] was run using 20Kb interval chunks of the input BED, padded by 100 bp on each side, such that if an indel occurred in between intervals, it would be captured.
+We ran Strelka2 [@doi:10/gdwrp4] v2.9.3  using default parameters for canonical chromosomes (chr1-22, X,Y,M), as recommended by the authors, and we filtered the final Strelka2 VCF for PASS variants.
+We ran Mutect2 from GATK v4.1.1.0 according to Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl], and we filtered the final Mutect2 VCF for PASS variants.
+To manage memory issues, we ran VarDictJava [@doi:10.1093/nar/gkw227] v1.58 [@url:https://github.com/AstraZeneca-NGS/VarDictJava] using 20 Kb interval chunks of the input BED, padded by 100 bp on each side, such that if an indel occurred in between intervals, it would be captured.
 Parameters and filtering followed [BCBIO standards](https://github.com/bcbio/bcbio-nextgen) except that variants with a variant allele frequency (VAF) >= 0.05 (instead of >= 0.10) were retained.
```

Comment:
Note - we used VAF in this section earlier, but here the acronym is defined. Acronym definition should occur at first usage in section,